Doc/whatsnew/3.13.rst

```diff
@@ -174,6 +174,17 @@ Other Language Changes
 
   (Contributed by Victor Stinner in :gh:`114570`.)
 
+* Allow controlling Expat >=2.6.0 reparse deferral (CVE-2023-52425)
+  by adding five new methods:
+
+  * :meth:`pyexpat.xmlparser.GetReparseDeferralEnabled`
+  * :meth:`pyexpat.xmlparser.SetReparseDeferralEnabled`
+  * :meth:`xml.etree.ElementTree.XMLParser.flush`
+  * :meth:`xml.etree.ElementTree.XMLPullParser.flush`
+  * :meth:`xml.sax.expatreader.ExpatParser.flush`
```

Comment:
```suggestion
  * :meth:`!xml.sax.expatreader.ExpatParser.flush`
```

Reply:
@serhiy-storchaka can you help me understand what the `!` does? Where can I read up on that syntax? Is this a vanilla rst thing?

Reply:
This is [standard reStructuredText syntax](https://www.sphinx-doc.org/en/master/usage/referencing.html). It means "do not create link". And Sphinx will not produce a warning "reference target not found". What's New should be warnings-free, but this method (and the whole class) is not documented, so it is the best that we can do now.

Reply:
@serhiy-storchaka that's great to know and makes good sense, thanks for explaining, done, pushed :+1